MyPiggyBank.Data/Model/CyclicOperation.cs

```diff
@@ -14,5 +14,7 @@ public class CyclicOperation
         public decimal EstimatedValue { get; set; }
         public string Period { get; set; }
```

Comment:
Zmiana typu ze string na int. W klasie biznesowej będzie on Enumem. W ten sposób bedzie to bardziej rygorystyczne. Nie pojawi się opcja, że cokolwiek się tam "dostanie".